src/transformState.js

```diff
@@ -6,6 +6,25 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const obj of actions) {
```

Comment:
Якщо з пакета яблук діставити один предмет по черзі то кожен такий предмет бути називатись яблуко а не obj

```suggestion
  for (const action of actions) {
```